server/src/dash-server.ts

```diff
@@ -112,8 +112,22 @@ export class DashServer {
       return;
     }
 
-    this.pushSubscriptions.addPushSubscription(
-        login, req.body.subscription, req.body.supportedContentEncodings);
+    switch (req.params.action) {
+      case 'add': {
```

Comment:
Personally not the biggest fan of a switch here. I think an if else would be less indentation, less lines and a tad more readable.